go/hashmodule/hashmodule.go

```diff
@@ -24,26 +24,32 @@ import (
 	"fmt"
 	"hash"
 
+	"github.com/spaolacci/murmur3"
+
 	"go.starlark.net/starlark"
 	"go.starlark.net/starlarkstruct"
 )
 
 // NewModule returns a Starlark module of common hash functions.
 //
-//  hash = module(
-//    md5,
-//    sha1,
-//    sha256,
-//  )
+// hash = module(
+//
+//	  md5,
+//		sha1,
+//		sha256,
+//		murmur3,
+//
+// )
 //
 // See `docs/modules.asciidoc` for details on the API of each function.
 func NewModule() *starlarkstruct.Module {
 	return &starlarkstruct.Module{
 		Name: "hash",
 		Members: starlark.StringDict{
-			"md5":    starlark.NewBuiltin("hash.md5", fnHash(md5.New)),
-			"sha1":   starlark.NewBuiltin("hash.sha1", fnHash(sha1.New)),
-			"sha256": starlark.NewBuiltin("hash.sha256", fnHash(sha256.New)),
+			"md5":        starlark.NewBuiltin("hash.md5", fnHash(md5.New)),
+			"sha1":       starlark.NewBuiltin("hash.sha1", fnHash(sha1.New)),
+			"sha256":     starlark.NewBuiltin("hash.sha256", fnHash(sha256.New)),
+			"murmur3_64": starlark.NewBuiltin("hash.murmur3_64", fnHash64(murmur3.New64)),
```

Comment:
`murmur3` above but `murmur3_64` here. Former seems better to me.

Reply:
fixed